src/applications/appeals/995/tests/fixtures/mocks/telephone-update.json

```diff
@@ -1,8 +1,8 @@
 {
   "data": {
     "attributes": {
-      "transaction_status": "RECEIVED",
-      "transaction_id": "c049d895-ecdf-40a4-ac0f-7947a06ea0c2",
+      "transactionStatus": "RECEIVED",
+      "transactionId": "c049d895-ecdf-40a4-ac0f-7947a06ea0c2",
```

Comment:
mock responses should be camelCased for the reducers for profile to work correctly.